### DISCUSSION TITLE
how to setup storage

### DISCUSSION THREAD

**Discussion by DaiZack** [apache/dolphinscheduler#10322] [Ideas]
I am using dock deployment. I am wondering how to set up file storage. It shows "storage not startup."

![image](https://user-images.githubusercontent.com/45176371/171260970-28b50c4f-2d5e-4e0e-bdb9-3cdb9b37e863.png)

I used profile all

![image](https://user-images.githubusercontent.com/45176371/171261242-d408e961-69d4-44f6-9d1a-2e53c6ad816c.png)

**Comment by EricGao888** [comment]
May I ask what kind of storage you configured? HDFS or LOCAL?

**Reply by DaiZack** [reply]
Local storage.

**Reply by EricGao888** [reply]
> Local storage.

Could you please provide a screenshot or paste your `common.properties`?

**Reply by DaiZack** [reply]
I don't have access to the server for now. Let me update this, when I get to it.

**Reply by EricGao888** [reply]
> Local storage.

@DaiZack You could wait for this PR https://github.com/apache/dolphinscheduler/pull/10265 merged. Or use a workaround. Set your storage type to `HDFS`, and set the `resource.hdfs.fs.defaultFS` as `file:///`

**Reply by DaiZack** [reply]
OK. Thank you very much.

**Reply by EricGao888** [reply]
> OK. Thank you very much.

You're welcome.
